osmomath/decimal.go

```diff
@@ -887,12 +887,16 @@ func MaxDec(d1, d2 BigDec) BigDec {
 
 // DecEq returns true if two given decimals are equal.
 // Intended to be used with require/assert:  require.True(t, DecEq(...))
+//
+//nolint:thelper
```

Comment:
What's the purpose of this though? I don't see CI complaining

Reply:
I did not get these warnings from CI, just saw them in my code editor and decided to fix them

Reply:
Alt: Can we edit your linter settings? 

Were going to get a nolintlint in the future I bet :)